### PR TITLE
Article on QGIS Server and OGC API Features

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

File: content/blog/ogcapif.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+by the OGC for the WMS 1.3.0 service (recently renewed for QGIS 3.10).
+
+<p align="center">
+<img src="/images/blog/ogcapif/badge.png" alt="alt wording" style="height:200px;">
+</p>
+
+Last year in 2019, a new protocol has been developped and named OGC API
+Features (commonly known as WFS3). With the purpose of having an up-to-date
+QGIS Server, the OSGeo have dedicated some funds to work on the implementation
+of this brand-new service. But we wanted to do it right, so the ambition was
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
of this brand-new service, but we wanted to do it right, so the ambition was
```

File: content/blog/ogcapif.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+also to reach the OGC certification!
+
+However, this new protocol gets rid of the XML specification to use the OpenAPI
+standard as well as the JSON open format instead. In other words, it's not just
+another protocol to support, but it's a whole package of changes and fresh
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
another protocol to support, but a whole package of changes and fresh
```

File: content/blog/ogcapif.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+mechanisms to work on. It was quite a challenge!
+
+At this time the QCooperative was remotely participated to OGC sprints to
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
At that time, the QCooperative was remotely participating in OGC sprints to
```

File: content/blog/ogcapif.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+closely monitor the development of the new OGC API Features protocol. Hence we
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
closely monitor the development of the new OGC API Features protocol. Hence, we
```

File: content/blog/ogcapif.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+started the implementation and a fully operationnal version landed in [QGIS
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
started its implementation and a fully operational version landed in [QGIS
```

File: content/blog/ogcapif.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+Server 3.10](http://blog.qgis.org/2019/11/26/qgis-server-is-ready-for-the-new-ogc-api-for-features-protocol/).
+
+
+# Implementation and features
+
+As a reminder, the WFS protocol allows to manipulate vector features unlike the
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
As a reminder, the WFS protocol allows to manipulate vector features, unlike the
```

File: content/blog/ogcapif.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+WMS format which provides raster outputs. OGC API Features is the natural
+continuity and consistently provides basic mechanisms to retieve features and
```
Comment (SrNetoChan) [inline content/blog/ogcapif.md]:
```suggestion
continuity and consistently provides basic mechanisms to retrieve features and
```

**Review by SrNetoChan — APPROVED** [review]
Made a few suggestions, but feel free to ignore them :-)

Thanks for the blog post.

Btw, do we really need the HTML blocks for images?

**Comment by pblottiere** [comment]
> Made a few suggestions, but feel free to ignore them :-)

Thanks! It's better now :)

> Btw, do we really need the HTML blocks for images?

I know of no other way for centering an image in markdown... Do you?

**Comment by SrNetoChan** [comment]
> I know of no other way for centering an image in markdown... Do you?

No, I don't think there is, but maybe it makes more sense to make all inages appear centere by changing the theme.

**Comment by pblottiere** [comment]
>> I know of no other way for centering an image in markdown... Do you?

> No, I don't think there is, but maybe it makes more sense to make all inages appear centere by changing the theme.

Mmm it seems that Hugo doesn't support the `=200x200` syntax to manage size of images in Markdown :/.

I don't like HTML block either, but in this particular case, it seems more convenient...

**Comment by SrNetoChan** [comment]
> > > I know of no other way for centering an image in markdown... Do you?
> 
> > No, I don't think there is, but maybe it makes more sense to make all inages appear centere by changing the theme.
> 
> Mmm it seems that Hugo doesn't support the `=200x200` syntax to manage size of images in Markdown :/.
> 
> I don't like HTML block either, but in this particular case, it seems more convenient...

Sorry for wasting your time.

**Comment by pblottiere** [comment]
> Sorry for wasting your time.

You didn't, it would have been nice to remove these HTML blocks, so it was worth investigating :).

**Comment by pblottiere** [comment]
@luipir I added your comment as a citation at the beginning of the article. It's simple but sounds like a friendly summary.

What do you think?

**Comment by luipir** [comment]
> @luipir I added your comment as a citation at the beginning of the article. It's simple but sounds like a friendly summary.
> 
> What do you think?

hahaha... it's ok for me and a friendly (the citation  not because it's referred to me) way to promote further reading

**Comment by pblottiere** [comment]
> hahaha... it's ok for me and a friendly (the citation not because it's referred to me) way to promote further reading

Good :).

I'll merge this PR tomorrow if I don't have more comments in the meantime.

**Pull request merged**